src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -88,6 +90,18 @@ public void parseCommand_list() throws Exception {
         assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD + " 3") instanceof ListCommand);
     }
 
+    @Test
+    public void parseCommand_paid() throws Exception {
+        assertTrue(parser.parseCommand("paid 1") instanceof PaidCommand);
+        assertTrue(parser.parseCommand("paid 2") instanceof PaidCommand);
+    }
+
+    @Test
+    public void parseCommand_unpaid() throws Exception {
+        assertTrue(parser.parseCommand("unpaid 1") instanceof UnpaidCommand);
+        assertTrue(parser.parseCommand("unpaid 2") instanceof UnpaidCommand);
+    }
+
```

Comment:
Well done adding relevant tests to ensure that AddressBook correctly reads newly added paid and unpaid commands.